.github/workflows/maven-release.yml

```diff
@@ -19,6 +19,9 @@
 name: Maven Release
 
 on: workflow_dispatch
+permissions:
```

Comment:
Just realized that we need to be able to `git push` here. So `contents:read` will not suffice.